Nos/Models/FlagOption.swift

```diff
@@ -8,29 +8,49 @@ import Foundation
 struct FlagOption: Identifiable {
     let title: String
     let description: String?
+    let info: String?
```

Comment:
It would be nice to have a comment describing how this will be used, because it wasn't obvious to me until I saw the code below. Something like "This is a message that will be displayed when the user has selected this Flag".

Reply:
Thanks for pointing that out @mplorentz